src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form action="http://mate-academy-form-lesson.herokuapp.com/creater-application">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div>
+          <label class="form-field">
+          Surname:
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 2nd item, fix your formatting of child elements everywhere
```suggestion
          <label class="form-field">
            Surname:
```